src/fn/fn-popover.scss

```diff
@@ -0,0 +1,136 @@
+@import "./fn-settings";
+
+$block: #{$fn-namespace}-popover;
+
+$fn-popover-arrow-size: 0.75rem;
+$fn-popover-padding: 0.75rem;
+$fn-popover-button-margin: 0.375rem;
+$fn-popover-arrow-offset: -0.25rem;
+$fn-popover-arrow-margin: 1.5rem;
+$fn-popover-arrow-middle-calc: calc(50% - 0.25rem);
+
+.#{$block} {
+  @include fn-reset();
+  @include fn-flex(column);
+
+  background-color: $fn-color-white;
+  border-radius: $fn-border-radius-12;
+  padding: $fn-popover-padding;
+  position: relative;
+  filter: drop-shadow(0 0 0.125rem rgba(0, 0, 0, 0.25)) drop-shadow(0 0.5rem 1rem rgba(0, 0, 0, 0.25));
+
+  &--no-padding {
+    padding: 0;
+  }
+
+  &__title {
+    @include fn-reset();
+
+    padding: $fn-popover-padding;
+    width: 100%;
+
+    .#{$block}__text {
+      @include fn-reset();
+
+      font-size: 1.125rem;
+      font-weight: bold;
+      line-height: 1.375rem;
+
+      &--truncate {
+        @include fn-ellipsis();
+      }
```

Comment:
Could you make one of the examples with long text so Chromatic detects future canges?